backend/infrahub/services/adapters/cache/redis.py

```diff
@@ -44,3 +50,7 @@ async def list_keys(self, filter_pattern: str) -> list[str]:
 
     async def set(self, key: str, value: str, expires: KVTTL | None = None, not_exists: bool = False) -> bool | None:
         return await self.connection.set(name=key, value=value, ex=expires.value if expires else None, nx=not_exists)
+
+    @classmethod
+    async def new(cls) -> RedisCache:
+        return cls()
```

Comment:
```suggestion
        return RedisCache()
```

Reply:
Nit pick: It would look clearer

Reply:
I'm not sure about replacing `cls` here I'd say it's pretty much standard in Python to initialize classmethods using `cls()`.